src/adaptors/http1/http1_private.h

```diff
@@ -268,4 +258,5 @@ void qdr_http1_server_core_conn_close(qdr_http1_adaptor_t *adaptor,
 void qdr_http1_record_client_request_info(qdr_http1_adaptor_t *adaptor, qdr_http1_request_base_t *request);
 void qdr_http1_record_server_request_info(qdr_http1_adaptor_t *adaptor, qdr_http1_request_base_t *request);
 
+#define HTTP1_READ_BUF_MAX 2
```

Comment:
HTTP1_READ_BUF_MAX is 2 so I guess the functions in adaptor_common.c cannot be used because they have RAW_BUFFER_BATCH set to 4 ?

Reply:
Oh, that's not longer used.  Good catch.